CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+0.4.3 (November 2018)
+-----------------
```

Comment:
minor: missing few ``--``
Could you fix that then we are good

Reply:
👍